src/blocks/remote-data-container/components/placeholders/ItemSelectQueryType.tsx

```diff
@@ -1,22 +1,29 @@
-import { Button, ButtonGroup } from '@wordpress/components';
+import {
+	Button,
+	__experimentalToggleGroupControl as ToggleGroupControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
 
 import { DataViewsModal } from '@/blocks/remote-data-container/components/modals/DataViewsModal';
 import { InputModal } from '@/blocks/remote-data-container/components/modals/InputModal';
 import { InputPopover } from '@/blocks/remote-data-container/components/popovers/InputPopover';
 
 interface ItemSelectQueryTypeProps {
-	blockConfig: BlockConfig;
+	blockName: string;
+	selectors: Selector[];
 	onSelect: ( data: RemoteDataQueryInput[] ) => void;
 }
 
 export function ItemSelectQueryType( props: ItemSelectQueryTypeProps ) {
-	const {
-		blockConfig: { name: blockName, selectors },
-		onSelect,
-	} = props;
+	const { blockName, selectors, onSelect } = props;
 
 	return (
-		<ButtonGroup className="remote-data-blocks-button-group">
+		<ToggleGroupControl
```

Comment:
I've been fixing all the deprecation warnings in the console, so this is part of that